manager/routes/routes.go

```diff
@@ -20,6 +20,7 @@ func InitAPI(api *gin.RouterGroup) {
 	baselines := api.Group("/baselines")
 	baselines.GET("/", controllers.BaselinesListHandler)
 	baselines.GET("/:baseline_id/systems", controllers.BaselineSystemsListHandler)
+	baselines.DELETE("/:baseline_id", controllers.BaselineDeleteHandler)
```

Comment:
I think write rbac permission should be used. So we need to update rbac.go.

Reply:
I believe we have already permission to delete: https://github.com/RedHatInsights/patchman-engine/blob/master/manager/middlewares/rbac.go#L77